config/locales/en.yml

```diff
@@ -255,11 +255,11 @@ en:
   title_none: None
   title_rejection: Rejection
   title_delegation: Delegation
-  title_assignment: Assignment
-  title_start: Start
+  title_assignment: Assign workflow
+  title_start: Start workflow
   title_dmsf_workflow_log: Approval Workflow Log
   title_assigned: Assigned
-  title_approval: Approval
+  title_approval: Approved
```

Comment:
After this change, a user would see "Approved" in the context menu of a document with a started (not yet approved) workflow. There is title_approved for "Approved" text. What is the reason for this change?

Reply:
Maybe you meant "Approve"?

Reply:
There is an inconsistency between German and English translation for the key `title_approval`:

```yaml
de:
  title_approval: Genehmigt

en:
  title_approval: Approval
````

Since `Genehmigt` translates to `Approved` I came up with this change. Both describes the **state** the workflow will get when clicking on the item in the context menu. And when the user approved the workflow, the menu item is confirming exactly this:

![image](https://github.com/danmunn/redmine_dmsf/assets/43378236/44d4911d-68e7-467c-aaaa-38db2b3b9024)


The english word `Approval` is not a state but a **substantive**. For me an adjective describing the state a workflow will get is the better choice.

Reply:
No, no, that's wrong. The workflow of the items in the menu should be as follows:
1. "Assign workflow"
2. "Start workflow"
3. "Approval" (Approve x Reject x Delegate)
4. "" (As soon as the workflow is finished, there is no item related to the workflow in the menu.)

Reply:
And how about this:

1.  "Assign workflow"
2. "Start workflow"
3. "Approve document" (Approve x Reject x Delegate)
4. "" (As soon as the workflow is finished, there is no item related to the workflow in the menu.)

Step 3 prompts the user similar as in steps 1 and 2. Looks even more consistent than my first proposal to me.

What do you think about it?

Reply:
Hm, I see as problematic that the word "Approve" suggests that you want to approve the document and might sound misleading when you want to reject it, actually.
But, let's go your way. Can you update your pull request accordingly?

Reply:
I understand your point. We can do the change and observe who it will feel to all other users ...

I will update the pull request soon.